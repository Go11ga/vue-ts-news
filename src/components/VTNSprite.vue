<template>
    <div style="display: none">

        <svg id="icon_list" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
            <rect width="18" height="8"/>
            <rect y="10" width="18" height="8"/>
        </svg>

        <svg id="icon_grid" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg">
            <rect width="8" height="8"/>
            <rect y="10" width="8" height="8"/>
            <rect x="10" width="8" height="8"/>
            <rect x="10" y="10" width="8" height="8"/>
        </svg>

    </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
    name: 'VTNSprite'
})
</script>
