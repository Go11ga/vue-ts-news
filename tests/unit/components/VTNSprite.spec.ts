import { shallowMount } from '@vue/test-utils'
import VTNSprite from '@/components/VTNSprite.vue'

describe('VTNSprite', () => {
    let wrapper = shallowMount(VTNSprite)

    it('dummy name test', () => {
        expect(wrapper.vm.$options.name).toMatch('VTNSprite')
    })

    it('snapshot', () => {
        expect(wrapper.element).toMatchSnapshot()
    })
})
