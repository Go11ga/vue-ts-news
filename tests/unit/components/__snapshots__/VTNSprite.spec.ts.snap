// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VTNSprite snapshot 1`] = `
<div
  style="display: none;"
>
  <svg
    id="icon_list"
    viewBox="0 0 18 18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      height="8"
      width="18"
    />
    <rect
      height="8"
      width="18"
      y="10"
    />
  </svg>
  <svg
    id="icon_grid"
    viewBox="0 0 18 18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <rect
      height="8"
      width="8"
    />
    <rect
      height="8"
      width="8"
      y="10"
    />
    <rect
      height="8"
      width="8"
      x="10"
    />
    <rect
      height="8"
      width="8"
      x="10"
      y="10"
    />
  </svg>
</div>
`;
